docs/custom_schemas/h5md/explanation-H5MD-NOMAD.md

```diff
@@ -0,0 +1,1342 @@
+    size 2, that contains the major version number and the minor version number
+    of the module.
+
+    The version *x.y.z* of an H5MD module follows [semantic versioning][semver]
+    [@semantic_versioning] and again only the components *x* and *y* are
+    stored, see `h5md/version` in "[H5MD metadata]."
+
+[semver]: http://semver.org/spec/v2.0.0.html -->
+
+
+
+
+## The particles group
+
+Particle attributes, i.e., information about each particle in the system, are stored within the `particles` group.
+According to the original H5MD schema, the `particles` group is a container for subgroups that
+represent different subsets of the system under consideration.
+For simplicity of parsing, H5MD-NOMAD currently requires one such group, labeled `all`, to contain all the particles and corresponding attributes to be stored in the NOMAD archive.
+**Additional particle groups will be ignored**.
+
+For each dataset, the ordering of indices (whenever relevant) is as follows: frame index, particle index, dimension index.
+Thus, the contents of the `particles` group for a trajectory with `N_frames` frames and `N_part` particles in a `D`-dimensional space can be represented:
+
+    particles
+     \-- all
+     |    \-- box
+     |    \-- (<time-dependent_vector_attribute>)
+     |    |    \-- step: Integer[N_frames]
+     |    |    \-- time: Float[N_frames]
+     |    |    \-- value: <type>[N_frames][N_part][D]
+     |    \-- (<time-dependent_scalar_attribute>)
+     |    |    \-- step: Integer[N_frames]
+     |    |    \-- time: Float[N_frames]
+     |    |    \-- value: <type>[N_frames][N_part]
+     |    \-- (<time-independent_vector_attribute>): <type>[N_part][D]
+     |    \-- (<time-independent_scalar_attribute>): <type>[N_part]
+     |    \-- ...
+     \-- <group2>
+          \-- ...
+
+### Standardized H5MD elements for particles group
+
+`position`
+:   **(required for parsing other particle attributes)** An element that describes the particle positions as coordinate vectors of `Float` or `Integer` type.
+
+<!-- If the component $k$ of `box/boundary` (see [below](#simulation-box)) is set
+to `none`, the data indicate for each particle the component $k$ of its
+absolute position in space. If the component $k$ of `box/boundary` is set to
+`periodic`, the data indicate for each particle the component $k$ of the
+absolute position in space of an *arbitrary* periodic image of that particle. -->
+
+`velocity`
+:   An element that contains the velocities for each particle as a vector of
+    `Float` or `Integer` type.
+
+`force`
+:   An element that contains the total forces (i.e., the accelerations
+    multiplied by the particle mass) for each particle as a vector of `Float`
+    or `Integer` type.
+
+`mass`
+:   An element that holds the mass for each particle as a scalar of `Float`
+    type.
+
+`image`
+:   <a id="image_anchor"></a>**(currently unused in H5MD-NOMAD)**
+
+??? details
```

Comment:
I like the idea of these admonitions, but the issue is that visually they are not so obviously linked with the quantity you are defined. I will keep it simply and add this description after the bold parenthesis, or directly, not adding any explanation.

Reply:
I made a TODO about this. I agree with your comment, but I really want this information to be hidden. I guess an alternative would be to link to the corresponding section of the original H5MD docs, but I'm not sure how persistent that is.